### PR TITLE
fixes angular-ui/bootstrap#4018

### PR DESCRIPTION
**Pull request by SteffanLong**

### CODE REVIEW

File: src/progressbar/progressbar.js [hunk 1]
```diff
@@ -27,6 +27,17 @@ angular.module('ui.bootstrap.progressbar', [])
...
             bar.percent = +(100 * bar.value / bar.max).toFixed(2);
+			
+            var totalPercentage = 0;
+            self.bars.forEach(function (bar) {
+                totalPercentage += bar.percent;
+            });
+
+            if (totalPercentage > 100) {
+                bar.percent -= totalPercentage - 100;
+            } else if (totalPercentage < 100) {
```
Comment (wesleycho) [inline src/progressbar/progressbar.js]: The `else if` should be removed - this is causing the tests to fail. We also should not assume that the bar should fill up the progress section.

**Comment by wesleycho** [comment]
Could you also write a test with three stacked bars, each with values that total 100% but round to greater than 100%?

**Comment by wesleycho** [comment]
Linking #4018 for reference.

**Comment by SteffanLong** [comment]
Sure thing.

File: src/progressbar/progressbar.js (hunk 1 above)
Comment (SteffanLong) [inline src/progressbar/progressbar.js]: Sorry, my mistake. I failed to realize the possible scenario of bars that do not reach the progress max. I'll remove that.

**Comment by SteffanLong** [comment]
I adjusted the code and added a test case.

**Comment by wesleycho** [comment]
This looks good - thanks for the good work here

**Pull request closed without merging**